setup.cfg

```diff
@@ -1,21 +0,0 @@
-[metadata]
-version = attr: simplemonitor.__version__
```

Comment:
this I'm not 100% sure on - I _think_ the static version in `pyproject.toml` `[project]` will be sufficient here?
From a quick look, I don't think that one is supposed to be dynamic, but should be hard-coded? I assume everything that needs to get updated already does as part of how it's released?

Reply:
I think so, but from memory this is also one of those things where I was trying a bunch of stuff out to see what worked for me and what didn't :)